tests/autoyast/installation.pm

```diff
@@ -228,6 +229,13 @@ sub run {
             die "installation ends in linuxrc";
         }
         elsif (match_has_tag('autoyast-confirm')) {
+            if (get_var('AY_INTERRUPT')) {
```

Comment:
we can add to this new test suite existing variable ENCRYPT_ACTIVATE_EXISTING so we don't need to create new one and have it documented. As discussed offline the interrupt doesn't look general solution, the sync is done by autoyast confirm.

Reply:
The problem is there are other teams using that module, if one has that variable, it will break, so I preferred to go for one that's clearly about pausing the install. I could replace it by PAUSE_INSTALL if you prefer.

Reply:
that name is better but we are the ones mainly testing ay and this scenario is kind of specific, please if you don't find such a case, let's go for not creating new variables (as our goal as a team is to get rid of them as much as possible).
Besides that thinking how this would scale, we could not use the pause for specific set of modules, so let's say if scenario 1 needs functions A and B conditionally to that unique setting for pause, how scenario 2 could run functions for A and C. Every function we would include there should have its own condition I think, so using existing should be better choice. Make sense?

Reply:
Makes sense, this module is used in qam-allpatterns and opensuse, but it would be really bad luck if they would have also that variable set somewhere, I think we're the only ones using it.